### ISSUE TITLE
User folder should create bookmarks folder for users

### ISSUE THREAD

**Issue by micahnyc** [OpenSpace/OpenSpace#3229]
Was with a user who was on v20 last night. We had to move their bookmarks file, but it wasn't clear where to put it. Then once we found out, we had a typo in the folder name we create so it still wanst working

Now that we have bookmarks in version 20 living in the user folder, we should create the /user/bookmarks folder for them, this will stop them from making any typos in the name when creating it themselves. In addition, when we refer to the folder in our documentation, it will exist even if empty for when users go to put something in there. 

It seems like we tried to do that here: https://github.com/OpenSpace/OpenSpace/blob/c09e6da4057798ae4d377c132994921d54fd0023/data/assets/global/localbookmarks.asset#L10 

but maybe its not creating the folder?

**Comment by engbergandreas** [comment]
Yep that line is just for checking if the file exists, makes sense to create the path as well.
